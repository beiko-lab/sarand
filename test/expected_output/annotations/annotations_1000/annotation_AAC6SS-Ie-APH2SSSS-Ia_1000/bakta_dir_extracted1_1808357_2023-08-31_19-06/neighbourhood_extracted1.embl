ID   contig_1; ; linear; DNA; ; UNC; 3440 BP.
XX
AC   contig_1;
XX
DE   contig_1, whole genome shotgun sequence
XX
OS   .
OC   .
XX
CC   Annotated with Bakta
CC   Software: v1.8.1
CC   Database: v5.0, light
CC   DOI: 10.1099/mgen.0.000685
CC   URL: github.com/oschwengers/bakta
CC   
CC   ##Genome Annotation Summary:##
CC   Annotation Date                :: 08/31/2023, 19:06:17
CC   CDSs                           ::     2
CC   tRNAs                          ::     0
CC   tmRNAs                         ::     0
CC   rRNAs                          ::     0
CC   ncRNAs                         ::     0
CC   regulatory ncRNAs              ::     0
CC   CRISPR Arrays                  ::     0
CC   oriCs/oriVs                    ::     0
CC   oriTs                          ::     0
CC   gaps                           ::     0
CC   pseudogenes                    ::     0
XX
FH   Key             Location/Qualifiers
FH
FT   source          1..3440
FT                   /mol_type="genomic DNA"
FT   gene            554..1000
FT                   /locus_tag="FKBKAJ_00005"
FT   CDS             554..1000
FT                   /db_xref="SO:0001217"
FT                   /db_xref="UniRef:UniRef50_Q7WTB9"
FT                   /product="N-acetyltransferase domain-containing protein"
FT                   /locus_tag="FKBKAJ_00005"
FT                   /protein_id="gnl|Bakta|FKBKAJ_00005"
FT                   /translation="MVKTLYSILHRSLDLTEFMEEVLIDDNMVFDIDNLKGFLNDTSSF
FT                   GFIAKENNKIIGFAYCYTLLRPDGKTMFYLHSIGMLPNYQDKGYGSKLLSFIKEYSKEI
FT                   GCSEMFLITDKGNPRACHVYEKLGGKNDYKDEIVYVYDYEKGDK"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT                   /inference="similar to AA sequence:UniRef:UniRef50_Q7WTB9"
FT   gene            1001..2440
FT                   /locus_tag="FKBKAJ_00010"
FT                   /gene="aac(6')-Ie"
FT   CDS             1001..2440
FT                   /db_xref="NCBIProtein:WP_001028144.1"
FT                   /db_xref="SO:0001217"
FT                   /db_xref="UniRef:UniRef50_P0A0C2"
FT                   /product="bifunctional aminoglycoside N-acetyltransferase
FT                   AAC(6')-Ie/aminoglycoside O-phosphotransferase APH(2'')-Ia"
FT                   /locus_tag="FKBKAJ_00010"
FT                   /protein_id="gnl|Bakta|FKBKAJ_00010"
FT                   /translation="MNIVENEICIRTLIDDDFPLMLKWLTDERVLEFYGGRDKKYTLES
FT                   LKKHYTEPWEDEVFRVIIEYNNVPIGYGQIYKMYDELYTDYHYPKTDEIVYGMDQFIGE
FT                   PNYWSKGIGTRYIKLIFEFLKKERNANAVILDPHKNNPRAIRAYQKSGFRIIEDLPEHE
FT                   LHEGKKEDCYLMEYRYDDNATNVKAMKYLIEHYFDNFKVDSIEIIGSGYDSVAYLVNNE
FT                   YIFKTKFSTNKKKGYAKEKAIYNFLNTNLETNVKIPNIEYSYISDELSILGYKEIKGTF
FT                   LTPEIYSTMSEEEQNLLKRDIASFLRQMHGLDYTDISECTIDNKQNVLEEYILLRETIY
FT                   NDLTDIEKDYIESFMERLNATTVFEGKKCLCHNDFSCNHLLLDGNNRLTGIIDFGDSGI
FT                   IDEYCDFIYLLEDSEEEIGTNFGEDILRMYGNIDIEKAKEYQDIVEEYYPIETIVYGIK
FT                   NIKQEFIENGRKEIYKRTYKD"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT                   /inference="similar to AA sequence:UniRef:UniRef50_P0A0C2"
FT                   /gene="aac(6')-Ie"
XX
SQ   Sequence 3440 BP; 1279 A; 404 C; 594 G; 1163 T; 0 other;
     aaaatacact caaaggcatt gaatgtattt acgctctata taaaaagaac cgcagatctc        60
     ttcagatcta cggattttcg ccatgccacg aaattaggga tatgttagcc agttaaggga       120
     acaacgacat gataaaatgg tagttagcta tatttttcta actttgcaac agaaccccta       180
     aaaaaaattc aaaatctttc agagaagctg ttaaaggaat ttttaagttc acagatatta       240
     acttagcgcg tgaggctaaa aatcgattga ttcatgatta tatcgatcaa ccaaaatatt       300
     caaaagcttg cgcatcattg gatgatggat tcgaagacgc ctttcaatat accgtacaag       360
     gaaattccca caatcgacta aagagtacca atctaattga acgactgaat caagaagtac       420
     gcagaagaga aaagattatt cgcatcttcc ccaatcaaac atcagccaat cgcttaattg       480
     gagccgttct tatggaccta catgatgaat ggatttattc ttcaagaaaa tacatcaatt       540
     ttgataagta gaaatggtaa aaacattgta tagcatttta cacaggagtc tggacttgac       600
     tgagtttatg gaagaagttt taattgatga taatatggtt tttgatattg ataatttaaa       660
     aggatttctt aatgatacca gttcatttgg gtttatagct aaagaaaata ataaaattat       720
     aggatttgca tattgctata cacttttaag acctgatgga aaaacaatgt tttatttaca       780
     ctcaatagga atgttaccta actatcaaga caaaggttat ggttcaaaat tattatcttt       840
     tattaaggaa tattctaaag agattggttg ttctgaaatg tttttaataa ctgataaagg       900
     taatcctaga gcttgccatg tatatgaaaa attaggtggt aaaaatgatt ataaagatga       960
     aatagtatat gtatatgatt atgaaaaagg tgataaataa atgaatatag ttgaaaatga      1020
     aatatgtata agaactttaa tagatgatga ttttcctttg atgttaaaat ggttaactga      1080
     tgaaagagta ttagaatttt atggtggtag agataaaaaa tatacattag aatcattaaa      1140
     aaaacattat acagagcctt gggaagatga agtttttaga gtaattattg aatataacaa      1200
     tgttcctatt ggatatggac aaatatataa aatgtatgat gagttatata ctgattatca      1260
     ttatccaaaa actgatgaga tagtctatgg tatggatcaa tttataggag agccaaatta      1320
     ttggagtaaa ggaattggta caagatatat taaattgatt tttgaatttt tgaaaaaaga      1380
     aagaaatgct aatgcagtta ttttagaccc tcataaaaat aatccaagag caataagggc      1440
     ataccaaaaa tctggtttta gaattattga agatttgcca gaacatgaat tacacgaggg      1500
     caaaaaagaa gattgttatt taatggaata tagatatgat gataatgcca caaatgttaa      1560
     ggcaatgaaa tatttaattg agcattactt tgataatttc aaagtagata gtattgaaat      1620
     aatcggtagt ggttatgata gtgtggcata tttagttaat aatgaataca tttttaaaac      1680
     aaaatttagt actaataaga aaaaaggtta tgcaaaagaa aaagcaatat ataatttttt      1740
     aaatacaaat ttagaaacta atgtaaaaat tcctaatatt gaatattcgt atattagtga      1800
     tgaattatct atactaggtt ataaagaaat taaaggaact tttttaacac cagaaattta      1860
     ttctactatg tcagaagaag aacaaaattt gttaaaacga gatattgcca gttttttaag      1920
     acaaatgcac ggtttagatt atacagatat tagtgaatgt actattgata ataaacaaaa      1980
     tgtattagaa gagtatatat tgttgcgtga aactatttat aatgatttaa ctgatataga      2040
     aaaagattat atagaaagtt ttatggaaag actaaatgca acaacagttt ttgagggtaa      2100
     aaagtgttta tgccataatg attttagttg taatcatcta ttgttagatg gcaataatag      2160
     attaactgga ataattgatt ttggagattc tggaattata gatgaatatt gtgattttat      2220
     atacttactt gaagatagtg aagaagaaat aggaacaaat tttggagaag atatattaag      2280
     aatgtatgga aatatagata ttgagaaagc aaaagaatat caagatatag ttgaagaata      2340
     ttatcctatt gaaactattg tttatggaat taaaaatatt aaacaggaat ttatcgaaaa      2400
     tggtagaaaa gaaatttata aaaggactta taaagattga ttatataata tatgaaaagc      2460
     tattataaaa gacattagta ttaaatagtt taaaaaaatg aaaaataata aaggaagtga      2520
     gtcaagtcca gactcctgtg taaaatgcta tacaatgttt ttaccatttc tacttatcaa      2580
     aattgatgta ttttcttgaa gaataaatcc attcatcatg taggtccata agaacggctc      2640
     caattaagcg attggctgat gtttgattgg ggaagatgcg aataatcttt tctcttctgc      2700
     gtacttcttg attcagtcgt tcaattagat tggtactctt tagtcgattg tgggaatttc      2760
     cttgtacggt atattgaaag gcgtcttcga atccatcatc caatgatgcg caagcttttg      2820
     aatattttgg ttgatcgata taatcatgaa tcaatcgatt tttagcctca cgcgctaagt      2880
     taatatctgt gaacttaaaa attcctttaa cagcttctct gaaagatttt gaattttttt      2940
     taggggttct gttgcaaagt tagaaaaata tagctaacta ccattttatc atgtcgttgt      3000
     tcccttaact ggctaacata tccctaattt cgtggcatgg cgaaaatccg tagatctgaa      3060
     gagatctgcg gttcttttta tatagagcgt aaatacattc aatgcctttg agtgtatttt      3120
     ttgccgtatt gatactttga tatcttgtct ttcttacttt aatatgacgg tgatcttgct      3180
     caatgaggtt attcagatat ttcgatgtac aatgacagtc aggtttaagt ttaaaagctt      3240
     taattacttt agctattgct actttcgttg aaggtgcctg atctgtaatt accttttgag      3300
     gtttaccaaa ttgtttaatg agacgtttga taaacgcata tgctgaatga ttatctcgtt      3360
     gcttacgcaa ccaaatatct aatgtatgtc cctctgcatc aatggcacga tataaatagc      3420
     tccattttcc ttttattttg                                                  3440
//
