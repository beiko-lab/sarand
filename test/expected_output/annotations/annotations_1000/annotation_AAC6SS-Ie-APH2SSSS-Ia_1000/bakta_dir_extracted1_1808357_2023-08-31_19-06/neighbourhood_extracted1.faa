>FKBKAJ_00005 N-acetyltransferase domain-containing protein
MVKTLYSILHRSLDLTEFMEEVLIDDNMVFDIDNLKGFLNDTSSFGFIAKENNKIIGFAYCYTLLRPDGKTMFYLHSIGMLPNYQDKGYGSKLLSFIKEYSKEIGCSEMFLITDKGNPRACHVYEKLGGKNDYKDEIVYVYDYEKGDK
>FKBKAJ_00010 bifunctional aminoglycoside N-acetyltransferase AAC(6')-Ie/aminoglycoside O-phosphotransferase APH(2'')-Ia
MNIVENEICIRTLIDDDFPLMLKWLTDERVLEFYGGRDKKYTLESLKKHYTEPWEDEVFRVIIEYNNVPIGYGQIYKMYDELYTDYHYPKTDEIVYGMDQFIGEPNYWSKGIGTRYIKLIFEFLKKERNANAVILDPHKNNPRAIRAYQKSGFRIIEDLPEHELHEGKKEDCYLMEYRYDDNATNVKAMKYLIEHYFDNFKVDSIEIIGSGYDSVAYLVNNEYIFKTKFSTNKKKGYAKEKAIYNFLNTNLETNVKIPNIEYSYISDELSILGYKEIKGTFLTPEIYSTMSEEEQNLLKRDIASFLRQMHGLDYTDISECTIDNKQNVLEEYILLRETIYNDLTDIEKDYIESFMERLNATTVFEGKKCLCHNDFSCNHLLLDGNNRLTGIIDFGDSGIIDEYCDFIYLLEDSEEEIGTNFGEDILRMYGNIDIEKAKEYQDIVEEYYPIETIVYGIKNIKQEFIENGRKEIYKRTYKD
