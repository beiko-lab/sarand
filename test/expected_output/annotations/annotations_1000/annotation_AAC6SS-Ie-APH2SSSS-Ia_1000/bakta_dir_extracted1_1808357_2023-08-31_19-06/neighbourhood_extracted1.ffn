>FKBKAJ_00005 N-acetyltransferase domain-containing protein
ATGGTAAAAACATTGTATAGCATTTTACACAGGAGTCTGGACTTGACTGAGTTTATGGAAGAAGTTTTAATTGATGATAATATGGTTTTTGATATTGATAATTTAAAAGGATTTCTTAATGATACCAGTTCATTTGGGTTTATAGCTAAAGAAAATAATAAAATTATAGGATTTGCATATTGCTATACACTTTTAAGACCTGATGGAAAAACAATGTTTTATTTACACTCAATAGGAATGTTACCTAACTATCAAGACAAAGGTTATGGTTCAAAATTATTATCTTTTATTAAGGAATATTCTAAAGAGATTGGTTGTTCTGAAATGTTTTTAATAACTGATAAAGGTAATCCTAGAGCTTGCCATGTATATGAAAAATTAGGTGGTAAAAATGATTATAAAGATGAAATAGTATATGTATATGATTATGAAAAAGGTGATAAATAA
>FKBKAJ_00010 bifunctional aminoglycoside N-acetyltransferase AAC(6')-Ie/aminoglycoside O-phosphotransferase APH(2'')-Ia
ATGAATATAGTTGAAAATGAAATATGTATAAGAACTTTAATAGATGATGATTTTCCTTTGATGTTAAAATGGTTAACTGATGAAAGAGTATTAGAATTTTATGGTGGTAGAGATAAAAAATATACATTAGAATCATTAAAAAAACATTATACAGAGCCTTGGGAAGATGAAGTTTTTAGAGTAATTATTGAATATAACAATGTTCCTATTGGATATGGACAAATATATAAAATGTATGATGAGTTATATACTGATTATCATTATCCAAAAACTGATGAGATAGTCTATGGTATGGATCAATTTATAGGAGAGCCAAATTATTGGAGTAAAGGAATTGGTACAAGATATATTAAATTGATTTTTGAATTTTTGAAAAAAGAAAGAAATGCTAATGCAGTTATTTTAGACCCTCATAAAAATAATCCAAGAGCAATAAGGGCATACCAAAAATCTGGTTTTAGAATTATTGAAGATTTGCCAGAACATGAATTACACGAGGGCAAAAAAGAAGATTGTTATTTAATGGAATATAGATATGATGATAATGCCACAAATGTTAAGGCAATGAAATATTTAATTGAGCATTACTTTGATAATTTCAAAGTAGATAGTATTGAAATAATCGGTAGTGGTTATGATAGTGTGGCATATTTAGTTAATAATGAATACATTTTTAAAACAAAATTTAGTACTAATAAGAAAAAAGGTTATGCAAAAGAAAAAGCAATATATAATTTTTTAAATACAAATTTAGAAACTAATGTAAAAATTCCTAATATTGAATATTCGTATATTAGTGATGAATTATCTATACTAGGTTATAAAGAAATTAAAGGAACTTTTTTAACACCAGAAATTTATTCTACTATGTCAGAAGAAGAACAAAATTTGTTAAAACGAGATATTGCCAGTTTTTTAAGACAAATGCACGGTTTAGATTATACAGATATTAGTGAATGTACTATTGATAATAAACAAAATGTATTAGAAGAGTATATATTGTTGCGTGAAACTATTTATAATGATTTAACTGATATAGAAAAAGATTATATAGAAAGTTTTATGGAAAGACTAAATGCAACAACAGTTTTTGAGGGTAAAAAGTGTTTATGCCATAATGATTTTAGTTGTAATCATCTATTGTTAGATGGCAATAATAGATTAACTGGAATAATTGATTTTGGAGATTCTGGAATTATAGATGAATATTGTGATTTTATATACTTACTTGAAGATAGTGAAGAAGAAATAGGAACAAATTTTGGAGAAGATATATTAAGAATGTATGGAAATATAGATATTGAGAAAGCAAAAGAATATCAAGATATAGTTGAAGAATATTATCCTATTGAAACTATTGTTTATGGAATTAAAAATATTAAACAGGAATTTATCGAAAATGGTAGAAAAGAAATTTATAAAAGGACTTATAAAGATTGA
