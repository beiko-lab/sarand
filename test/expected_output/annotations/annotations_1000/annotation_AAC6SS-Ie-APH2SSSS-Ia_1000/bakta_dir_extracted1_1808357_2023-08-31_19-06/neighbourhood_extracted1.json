{
    "genome": {
        "genus": null,
        "species": null,
        "strain": null,
        "complete": false,
        "gram": "?",
        "translation_table": 11
    },
    "stats": {
        "no_sequences": 1,
        "size": 3440,
        "gc": 0.29011627906976745,
        "n_ratio": 0.0,
        "n50": 3440,
        "coding_ratio": 0.548546511627907
    },
    "features": [
        {
            "type": "cds",
            "contig": "contig_1",
            "start": 554,
            "stop": 1000,
            "strand": "+",
            "gene": null,
            "product": "N-acetyltransferase domain-containing protein",
            "start_type": "ATG",
            "rbs_motif": null,
            "db_xrefs": [
                "SO:0001217",
                "UniRef:UniRef50_Q7WTB9"
            ],
            "frame": 2,
            "aa": "MVKTLYSILHRSLDLTEFMEEVLIDDNMVFDIDNLKGFLNDTSSFGFIAKENNKIIGFAYCYTLLRPDGKTMFYLHSIGMLPNYQDKGYGSKLLSFIKEYSKEIGCSEMFLITDKGNPRACHVYEKLGGKNDYKDEIVYVYDYEKGDK",
            "aa_hexdigest": "d454554ca51236f50f79e4a41f8ce178",
            "nt": "ATGGTAAAAACATTGTATAGCATTTTACACAGGAGTCTGGACTTGACTGAGTTTATGGAAGAAGTTTTAATTGATGATAATATGGTTTTTGATATTGATAATTTAAAAGGATTTCTTAATGATACCAGTTCATTTGGGTTTATAGCTAAAGAAAATAATAAAATTATAGGATTTGCATATTGCTATACACTTTTAAGACCTGATGGAAAAACAATGTTTTATTTACACTCAATAGGAATGTTACCTAACTATCAAGACAAAGGTTATGGTTCAAAATTATTATCTTTTATTAAGGAATATTCTAAAGAGATTGGTTGTTCTGAAATGTTTTTAATAACTGATAAAGGTAATCCTAGAGCTTGCCATGTATATGAAAAATTAGGTGGTAAAAATGATTATAAAGATGAAATAGTATATGTATATGATTATGAAAAAGGTGATAAATAA",
            "pscc": {
                "uniref50_id": "UniRef50_Q7WTB9",
                "db_xrefs": [
                    "SO:0001217",
                    "UniRef:UniRef50_Q7WTB9"
                ],
                "product": "N-acetyltransferase domain-containing protein"
            },
            "genes": [],
            "id": "FKBKAJFLFC_1",
            "locus": "FKBKAJ_00005"
        },
        {
            "type": "cds",
            "contig": "contig_1",
            "start": 1001,
            "stop": 2440,
            "strand": "+",
            "gene": "aac(6')-Ie",
            "product": "bifunctional aminoglycoside N-acetyltransferase AAC(6')-Ie/aminoglycoside O-phosphotransferase APH(2'')-Ia",
            "start_type": "ATG",
            "rbs_motif": "GGA/GAG/AGG",
            "db_xrefs": [
                "NCBIProtein:WP_001028144.1",
                "SO:0001217",
                "UniRef:UniRef50_P0A0C2"
            ],
            "frame": 2,
            "aa": "MNIVENEICIRTLIDDDFPLMLKWLTDERVLEFYGGRDKKYTLESLKKHYTEPWEDEVFRVIIEYNNVPIGYGQIYKMYDELYTDYHYPKTDEIVYGMDQFIGEPNYWSKGIGTRYIKLIFEFLKKERNANAVILDPHKNNPRAIRAYQKSGFRIIEDLPEHELHEGKKEDCYLMEYRYDDNATNVKAMKYLIEHYFDNFKVDSIEIIGSGYDSVAYLVNNEYIFKTKFSTNKKKGYAKEKAIYNFLNTNLETNVKIPNIEYSYISDELSILGYKEIKGTFLTPEIYSTMSEEEQNLLKRDIASFLRQMHGLDYTDISECTIDNKQNVLEEYILLRETIYNDLTDIEKDYIESFMERLNATTVFEGKKCLCHNDFSCNHLLLDGNNRLTGIIDFGDSGIIDEYCDFIYLLEDSEEEIGTNFGEDILRMYGNIDIEKAKEYQDIVEEYYPIETIVYGIKNIKQEFIENGRKEIYKRTYKD",
            "aa_hexdigest": "fd2a1e4266c3617f087b8f2bca65e616",
            "nt": "ATGAATATAGTTGAAAATGAAATATGTATAAGAACTTTAATAGATGATGATTTTCCTTTGATGTTAAAATGGTTAACTGATGAAAGAGTATTAGAATTTTATGGTGGTAGAGATAAAAAATATACATTAGAATCATTAAAAAAACATTATACAGAGCCTTGGGAAGATGAAGTTTTTAGAGTAATTATTGAATATAACAATGTTCCTATTGGATATGGACAAATATATAAAATGTATGATGAGTTATATACTGATTATCATTATCCAAAAACTGATGAGATAGTCTATGGTATGGATCAATTTATAGGAGAGCCAAATTATTGGAGTAAAGGAATTGGTACAAGATATATTAAATTGATTTTTGAATTTTTGAAAAAAGAAAGAAATGCTAATGCAGTTATTTTAGACCCTCATAAAAATAATCCAAGAGCAATAAGGGCATACCAAAAATCTGGTTTTAGAATTATTGAAGATTTGCCAGAACATGAATTACACGAGGGCAAAAAAGAAGATTGTTATTTAATGGAATATAGATATGATGATAATGCCACAAATGTTAAGGCAATGAAATATTTAATTGAGCATTACTTTGATAATTTCAAAGTAGATAGTATTGAAATAATCGGTAGTGGTTATGATAGTGTGGCATATTTAGTTAATAATGAATACATTTTTAAAACAAAATTTAGTACTAATAAGAAAAAAGGTTATGCAAAAGAAAAAGCAATATATAATTTTTTAAATACAAATTTAGAAACTAATGTAAAAATTCCTAATATTGAATATTCGTATATTAGTGATGAATTATCTATACTAGGTTATAAAGAAATTAAAGGAACTTTTTTAACACCAGAAATTTATTCTACTATGTCAGAAGAAGAACAAAATTTGTTAAAACGAGATATTGCCAGTTTTTTAAGACAAATGCACGGTTTAGATTATACAGATATTAGTGAATGTACTATTGATAATAAACAAAATGTATTAGAAGAGTATATATTGTTGCGTGAAACTATTTATAATGATTTAACTGATATAGAAAAAGATTATATAGAAAGTTTTATGGAAAGACTAAATGCAACAACAGTTTTTGAGGGTAAAAAGTGTTTATGCCATAATGATTTTAGTTGTAATCATCTATTGTTAGATGGCAATAATAGATTAACTGGAATAATTGATTTTGGAGATTCTGGAATTATAGATGAATATTGTGATTTTATATACTTACTTGAAGATAGTGAAGAAGAAATAGGAACAAATTTTGGAGAAGATATATTAAGAATGTATGGAAATATAGATATTGAGAAAGCAAAAGAATATCAAGATATAGTTGAAGAATATTATCCTATTGAAACTATTGTTTATGGAATTAAAAATATTAAACAGGAATTTATCGAAAATGGTAGAAAAGAAATTTATAAAAGGACTTATAAAGATTGA",
            "pscc": {
                "uniref50_id": "UniRef50_P0A0C2",
                "db_xrefs": [
                    "SO:0001217",
                    "UniRef:UniRef50_P0A0C2"
                ],
                "product": "Bifunctional AAC/APH"
            },
            "expert": [
                {
                    "type": "amrfinder",
                    "rank": 95,
                    "gene": "aac(6')-Ie/aph(2'')-Ia",
                    "product": "bifunctional aminoglycoside N-acetyltransferase AAC(6')-Ie/aminoglycoside O-phosphotransferase APH(2'')-Ia",
                    "method": "EXACTP",
                    "query_cov": 1.0,
                    "model_cov": 1.0,
                    "identity": 1.0,
                    "id": "WP_001028144.1",
                    "db_xrefs": [
                        "NCBIProtein:WP_001028144.1"
                    ]
                }
            ],
            "genes": [
                "aac(6')-Ie",
                "aph(2'')-Ia"
            ],
            "id": "FKBKAJFLFC_2",
            "locus": "FKBKAJ_00010"
        }
    ],
    "sequences": [
        {
            "id": "contig_1",
            "description": "[gcode=11] [topology=linear]",
            "sequence": "AAAATACACTCAAAGGCATTGAATGTATTTACGCTCTATATAAAAAGAACCGCAGATCTCTTCAGATCTACGGATTTTCGCCATGCCACGAAATTAGGGATATGTTAGCCAGTTAAGGGAACAACGACATGATAAAATGGTAGTTAGCTATATTTTTCTAACTTTGCAACAGAACCCCTAAAAAAAATTCAAAATCTTTCAGAGAAGCTGTTAAAGGAATTTTTAAGTTCACAGATATTAACTTAGCGCGTGAGGCTAAAAATCGATTGATTCATGATTATATCGATCAACCAAAATATTCAAAAGCTTGCGCATCATTGGATGATGGATTCGAAGACGCCTTTCAATATACCGTACAAGGAAATTCCCACAATCGACTAAAGAGTACCAATCTAATTGAACGACTGAATCAAGAAGTACGCAGAAGAGAAAAGATTATTCGCATCTTCCCCAATCAAACATCAGCCAATCGCTTAATTGGAGCCGTTCTTATGGACCTACATGATGAATGGATTTATTCTTCAAGAAAATACATCAATTTTGATAAGTAGAAATGGTAAAAACATTGTATAGCATTTTACACAGGAGTCTGGACTTGACTGAGTTTATGGAAGAAGTTTTAATTGATGATAATATGGTTTTTGATATTGATAATTTAAAAGGATTTCTTAATGATACCAGTTCATTTGGGTTTATAGCTAAAGAAAATAATAAAATTATAGGATTTGCATATTGCTATACACTTTTAAGACCTGATGGAAAAACAATGTTTTATTTACACTCAATAGGAATGTTACCTAACTATCAAGACAAAGGTTATGGTTCAAAATTATTATCTTTTATTAAGGAATATTCTAAAGAGATTGGTTGTTCTGAAATGTTTTTAATAACTGATAAAGGTAATCCTAGAGCTTGCCATGTATATGAAAAATTAGGTGGTAAAAATGATTATAAAGATGAAATAGTATATGTATATGATTATGAAAAAGGTGATAAATAAATGAATATAGTTGAAAATGAAATATGTATAAGAACTTTAATAGATGATGATTTTCCTTTGATGTTAAAATGGTTAACTGATGAAAGAGTATTAGAATTTTATGGTGGTAGAGATAAAAAATATACATTAGAATCATTAAAAAAACATTATACAGAGCCTTGGGAAGATGAAGTTTTTAGAGTAATTATTGAATATAACAATGTTCCTATTGGATATGGACAAATATATAAAATGTATGATGAGTTATATACTGATTATCATTATCCAAAAACTGATGAGATAGTCTATGGTATGGATCAATTTATAGGAGAGCCAAATTATTGGAGTAAAGGAATTGGTACAAGATATATTAAATTGATTTTTGAATTTTTGAAAAAAGAAAGAAATGCTAATGCAGTTATTTTAGACCCTCATAAAAATAATCCAAGAGCAATAAGGGCATACCAAAAATCTGGTTTTAGAATTATTGAAGATTTGCCAGAACATGAATTACACGAGGGCAAAAAAGAAGATTGTTATTTAATGGAATATAGATATGATGATAATGCCACAAATGTTAAGGCAATGAAATATTTAATTGAGCATTACTTTGATAATTTCAAAGTAGATAGTATTGAAATAATCGGTAGTGGTTATGATAGTGTGGCATATTTAGTTAATAATGAATACATTTTTAAAACAAAATTTAGTACTAATAAGAAAAAAGGTTATGCAAAAGAAAAAGCAATATATAATTTTTTAAATACAAATTTAGAAACTAATGTAAAAATTCCTAATATTGAATATTCGTATATTAGTGATGAATTATCTATACTAGGTTATAAAGAAATTAAAGGAACTTTTTTAACACCAGAAATTTATTCTACTATGTCAGAAGAAGAACAAAATTTGTTAAAACGAGATATTGCCAGTTTTTTAAGACAAATGCACGGTTTAGATTATACAGATATTAGTGAATGTACTATTGATAATAAACAAAATGTATTAGAAGAGTATATATTGTTGCGTGAAACTATTTATAATGATTTAACTGATATAGAAAAAGATTATATAGAAAGTTTTATGGAAAGACTAAATGCAACAACAGTTTTTGAGGGTAAAAAGTGTTTATGCCATAATGATTTTAGTTGTAATCATCTATTGTTAGATGGCAATAATAGATTAACTGGAATAATTGATTTTGGAGATTCTGGAATTATAGATGAATATTGTGATTTTATATACTTACTTGAAGATAGTGAAGAAGAAATAGGAACAAATTTTGGAGAAGATATATTAAGAATGTATGGAAATATAGATATTGAGAAAGCAAAAGAATATCAAGATATAGTTGAAGAATATTATCCTATTGAAACTATTGTTTATGGAATTAAAAATATTAAACAGGAATTTATCGAAAATGGTAGAAAAGAAATTTATAAAAGGACTTATAAAGATTGATTATATAATATATGAAAAGCTATTATAAAAGACATTAGTATTAAATAGTTTAAAAAAATGAAAAATAATAAAGGAAGTGAGTCAAGTCCAGACTCCTGTGTAAAATGCTATACAATGTTTTTACCATTTCTACTTATCAAAATTGATGTATTTTCTTGAAGAATAAATCCATTCATCATGTAGGTCCATAAGAACGGCTCCAATTAAGCGATTGGCTGATGTTTGATTGGGGAAGATGCGAATAATCTTTTCTCTTCTGCGTACTTCTTGATTCAGTCGTTCAATTAGATTGGTACTCTTTAGTCGATTGTGGGAATTTCCTTGTACGGTATATTGAAAGGCGTCTTCGAATCCATCATCCAATGATGCGCAAGCTTTTGAATATTTTGGTTGATCGATATAATCATGAATCAATCGATTTTTAGCCTCACGCGCTAAGTTAATATCTGTGAACTTAAAAATTCCTTTAACAGCTTCTCTGAAAGATTTTGAATTTTTTTTAGGGGTTCTGTTGCAAAGTTAGAAAAATATAGCTAACTACCATTTTATCATGTCGTTGTTCCCTTAACTGGCTAACATATCCCTAATTTCGTGGCATGGCGAAAATCCGTAGATCTGAAGAGATCTGCGGTTCTTTTTATATAGAGCGTAAATACATTCAATGCCTTTGAGTGTATTTTTTGCCGTATTGATACTTTGATATCTTGTCTTTCTTACTTTAATATGACGGTGATCTTGCTCAATGAGGTTATTCAGATATTTCGATGTACAATGACAGTCAGGTTTAAGTTTAAAAGCTTTAATTACTTTAGCTATTGCTACTTTCGTTGAAGGTGCCTGATCTGTAATTACCTTTTGAGGTTTACCAAATTGTTTAATGAGACGTTTGATAAACGCATATGCTGAATGATTATCTCGTTGCTTACGCAACCAAATATCTAATGTATGTCCCTCTGCATCAATGGCACGATATAAATAGCTCCATTTTCCTTTTATTTTG",
            "length": 3440,
            "complete": false,
            "type": "contig",
            "topology": "linear",
            "simple_id": "contig_1",
            "orig_id": "sequence:",
            "orig_description": "sequence:"
        }
    ],
    "run": {
        "start": "2023-08-31 19:06:07",
        "end": "2023-08-31 19:06:17"
    },
    "version": {
        "bakta": "1.8.1",
        "db": {
            "version": "5.0",
            "type": "light"
        }
    }
}